src/test/java/org/apache/maven/shared/utils/cli/CommandLineUtilsTest.java

```diff
@@ -128,33 +127,37 @@ public void givenASingleQuoteMarkInArgument_whenExecutingCode_thenExitCode0Retur
     }
 
     @Test
-    public void givenASingleQuoteMarkInArgument_whenTranslatingToCmdLineArgs_thenTheQuotationMarkIsNotEscaped() throws Exception
+    public void givenASingleQuoteMarkInArgument_whenTranslatingToCmdLineArgs_thenTheQuotationMarkIsNotEscaped()
+        throws Exception
     {
         final String command = "echo \"let's go\"";
-        final String[] expected = new String[]{"echo", "let's go"};
-        assertCmdLineArgs(expected, command);
+        final String[] expected = new String[] { "echo", "let's go" };
+        assertCmdLineArgs( expected, command );
     }
 
     @Test
-    public void givenAnEscapedDoubleQuoteMarkInArgument_whenTranslatingToCmdLineArgs_thenTheQuotationMarkRemainsEscaped() throws Exception
+    public void givenAnEscapedDoubleQuoteMarkInArgument_whenTranslatingToCmdLineArgs_thenTheQuotationMarkRemainsEscaped()
+        throws Exception
     {
         final String command = "echo \"let\\\"s go\"";
-        final String[] expected = new String[]{"echo", "let\\\"s go"};
-        assertCmdLineArgs(expected, command);
+        final String[] expected = new String[] { "echo", "let\\\"s go" };
+        assertCmdLineArgs( expected, command );
     }
 
     @Test
-    public void givenAnEscapedSingleQuoteMarkInArgument_whenTranslatingToCmdLineArgs_thenTheQuotationMarkRemainsEscaped() throws Exception
+    public void givenAnEscapedSingleQuoteMarkInArgument_whenTranslatingToCmdLineArgs_thenTheQuotationMarkRemainsEscaped()
+        throws Exception
     {
         final String command = "echo \"let\\\'s go\"";
-        final String[] expected = new String[]{"echo", "let\\\'s go"};
-        assertCmdLineArgs(expected, command);
+        final String[] expected = new String[] { "echo", "let\\\'s go" };
```

Comment:
Why people use `final` in such trivial cases?
I understand the reason to use it due to anonymous classes or immutable objects.

Reply:
The world seems split on this one. I personally avoid final on local variables except where it's required, mostly because that's my employer's style; but in Maven I've been trying to leave `final` in place where it already exists to avoid excess churn.

Reply:
this commit is now just about code formatting, so I'm fine with this change